src/main/java/software/amazon/encryption/s3/internal/ReEncryptInstructionFileResponse.java

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.amazon.encryption.s3.internal;
+
+/**
+ * Response object returned after re-encrypting an instruction file in S3.
+ * Contains the S3 bucket name, object key, and instruction file suffix used for the re-encrypted instruction file
+ */
+public class ReEncryptInstructionFileResponse {
+  private final String bucket;
+  private final String key;
+  private final String instructionFileSuffix;
+
+  /**
+   * Creates a new ReEncryptInstructionFileResponse object with the specified parameters.
+   *
+   * @param bucket the S3 bucket containing the re-encrypted instruction file
+   * @param key the S3 object key of the encrypted object in S3
+   * @param instructionFileSuffix the suffix used for the instruction file
+   */
+  public ReEncryptInstructionFileResponse(String bucket, String key, String instructionFileSuffix) {
+    this.bucket = bucket;
+    this.key = key;
+    this.instructionFileSuffix = instructionFileSuffix;
+  }
+
+  /**
+   * @return the S3 bucket containing the re-encrypted instruction file
+   */
+  public String Bucket() {
```

Comment:
nit: the request uses lowercase methods, let's be consistent and use lowercase here too.